.github/workflows/build-fast.yml

```diff
@@ -90,5 +90,56 @@ jobs:
         if: ${{!cancelled()}}
         run: |
           ccache -s
+  windows:
+    env:
+      CCACHE_DIR: "${{github.workspace}}\\.ccache"
+      CCACHE_MAXSIZE: "100Mi"
+      CMAKE_PRESET: fast
+    runs-on: windows-latest
+    steps:
+      - name: Install dependencies
+        run: |
+          choco install ninja ccache
+      - name: Check out Celeritas
+        uses: actions/checkout@v4
+      - name: Set up MSVC
+        uses: ilammy/msvc-dev-cmd@v1
+      - name: Cache ccache
+        uses: actions/cache@v4
+        with:
+          path: ${{env.CCACHE_DIR}}
+          key: ccache-fast-windows-${{github.run_id}}
+          restore-keys: |
+            ccache-fast
+      - name: Zero ccache stats
+        run: |
+          ccache -z
+      - name: Configure Celeritas
+        shell: pwsh
+        run: |
+          Copy-Item scripts/cmake-presets/ci-windows-github.json -Destination CMakeUserPresets.json
+          cmake --preset=$Env:CMAKE_PRESET `
+            -DCeleritas_GIT_DESCRIBE="${{github.event.pull_request
+              && format(';-pr.{0};', github.event.pull_request.number)
+              || format(';-{0};', github.ref_name)}}"
+      - name: Build all
+        run: |
+          cmake --build --preset=$Env:CMAKE_PRESET 
+      - name: Test all
+        continue-on-error: true
+        run: |
+          ctest --preset=$Env:CMAKE_PRESET 
```

Comment:
This seems to expand properly:
```
ctest --preset=$Env:CMAKE_PRESET 
```
and the test are not actually run (likely because of the missing config flag).

Reply:
The tests are run, and I think it's expanding correctly; this is the windows way of doing it. The failing tests didn't kill the job because I had `continue-on-error` set to true.

Reply:
In the run I looked at, they had not.  I see that indeed are running now.